src/types/vendor/@metamask/jazzicon.d.ts

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/unambiguous
+declare module '@metamask/jazzicon';
```

Comment:
Can you add more complete types?

Reply:
may i ask what does it mean by complete types?

Reply:
noted

Reply:
updated